tests/Unit/Passport/RoutesTest.php

```diff
@@ -56,6 +56,7 @@ public function testDontGetAccessTokenWithBadClientId()
 
         $this->assertEquals(json_decode($response, true), [
             'error' => 'invalid_client',
+            'error_description' => 'Client authentication failed',
```

Comment:
Interesting, is the addition of description for client message flexibility?

Reply:
Again, it just appeared.

Reply:
Worth looking at the Passport docs to note intent? I guess by default the field copies whatever we use in message. The change might mena our implementation isn't using a new feature/ improvement.

Reply:
As your instinct goes though... we probably don;t want to get too stuck on anything trivial in terms of time / money spent. So your strategy is fair!

Reply:
agreed